client/views/app/room.html

```diff
@@ -147,6 +147,7 @@
 											<p><i class="icon-phone"></i> {{phone}}</p>
 										{{/if}}
 										--}}
+                                                                            <span class="glyphicon-heart" title="Group Owner"></span>
```

Comment:
should there be a isOwner check before displaying this?  Also we used a check in the member list, but a heart here for the icon